_posts/2018-02-02-upgrade-rails-from-4-2-to-5-0.markdown

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Upgrade Rails from 4.2 to 5.0"
+date: 2018-03-01 11:19:00
+categories: ["rails", "upgrades"]
+author: "mauro-oto"
+---
+
+_This article is part of our Upgrade Rails series. To see more of them, [click here](https://www.ombulabs.com/blog/tags/upgrades)_.
+
+This article will cover the most important aspects that you need to know to get
+your [Ruby on Rails](http://rubyonrails.org/) application from [version 4.2](http://guides.rubyonrails.org/4_2_release_notes.html) to [5.0](http://guides.rubyonrails.org/5_0_release_notes.html).
+
+<!--more-->
+
+1. [Ruby version](#ruby-version)
+2. [Gems](#gems)
+3. [Config files (config/)](#config-files)
+4. [Application code](#application-code)
+  1. [ActiveRecord](#active-record)
+  2. [Controllers](#controllers)
+5. [Testing](#testing)
+6. [Next steps](#next-steps)
+
+<h2 id="ruby-version">1. Ruby version</h2>
+
+[Rails 5.0](http://weblog.rubyonrails.org/2016/6/30/Rails-5-0-final/) requires [Ruby 2.2.2](https://www.ruby-lang.org/en/news/2015/04/13/ruby-2-2-2-released/) or later.
+
+This Ruby upgrade shouldn't generate any problems. However, if you run into this
+exception in your test suite:
+
+`cannot load such file -- test/unit/assertions (Load Error)`
+
+then you'll need to add:
+
+`gem 'test-unit'`
+
+to your `Gemfile`.
+
+<h2 id="gems">2. Gems</h2>
+
+- It's recommended that you check your `Gemfile` against [Ready4Rails](http://www.ready4rails.net)
+to check whether you have any older gems which may not be compatible with Rails
+5 yet. (As of the release of this blog post, there are only a few gems which
+don't support Rails 5 yet, this is more of a problem when you're upgrading
+early on).
+
+If any of the gems are missing on Ready4Rails, you'll need to check the Github
+page for the project manually. In case you own the gem, you'll need to make sure
```

Comment:
@mauro-oto I think it sounds better if you saw "you will need to check the Github page for the project one by one" or "you will need to manually check the Github page for the project"